cli/script/command-executor.ts

```diff
@@ -26,14 +26,14 @@ var progress = require("progress");
 var configFilePath: string = path.join(process.env.LOCALAPPDATA || process.env.HOME, ".code-push.config");
 var userAgent: string = packageJson.name + "/" + packageJson.version;
```

Comment:
This sets the user-agent to be `code-push-cli/<CLI-version>`, whereas the default SDK user agent is `code-push/<SDK-version>`. I think it's useful to distinguish the CLI as a distinct user agent, especially when the SDK will be consumed externally. But, I can also see the simplicity benefit of having user-agent strings being defined *only* by the SDK version, so feel free to discuss if you think that should be the case!